e2e/js_image/pnpm-workspace.yaml

```diff
@@ -0,0 +1,2 @@
+packages:
```

Comment:
why add these files?

Reply:
pnpm does a funny thing when you call pnpm install. If it finds a workspace file anywhere up the tree then it does the install for that workspace instead of your current working directory. Simple workspace file with just the root package in subdirectories solves the problem.

Reply:
Interesting. I guess it doesn't stop when it sees a package.json or lockfile then? 🤔 

It might be nice putting something like that in its own commit with a message saying that 🤷

Reply:
or maybe add `e2e/pnpm-workspace.yaml` with empty or broken content ?

Reply:
as it is, this makes it so we don't have any e2e coverage for non-workspace case

Reply:
lock file generated when there isn't a workspace.yaml and when there is one with just a `.` package is identical. No workspace file basically implies a workspaces with a single importer at the root.

Reply:
but yeah, I'll break those new files out to a new PR and we can take the discussion there